canonical/mimir-k8s-operator
# add support for `metrics-endpoint` provider

## Issue
The Mimir charm is currently missing the `metrics-endpoint` provider which would allow it to be scraped by Prometheus.


## Solution
Implement the relation provider in Mimir.


## Testing Instructions
Deploy Mimir and Prometheus, relate them over the new interface and verify that Prometheus receives the metrics correctly.
